tool/site/server/src/routes/api/convert.ts

```diff
@@ -0,0 +1,23 @@
+import Hapi from "@hapi/hapi"
+import * as fhir from "fhir/r4"
+import {getEpsClient} from "../../services/communication/eps-client"
+import {getApigeeAccessTokenFromSession} from "../../services/session"
+
+export default [
+  {
+    method: "POST",
+    path: "/api/convert",
+    handler: async (request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit): Promise<Hapi.ResponseObject> => {
+      const bundle = request.payload as fhir.Bundle
```

Comment:
do we want to maybe keep this more generic i.e. `as fhir.Resource` so we can convert other requests down the line?